core/domain/stats_jobs_continuous.py

```diff
@@ -132,16 +132,6 @@ def reduce(key, stringified_values):
             for value_dict in value_dicts]))
         versions.sort(reverse=True)
 
-        # For answers mapped to specific versions, the versions list should only
-        # contain the version they correspond to. Otherwise, if they map to
-        # VERSION_ALL, then multiple versions may be included.
-        if exploration_version != VERSION_ALL and (
-                len(versions) != 1 or versions[0] != int(exploration_version)):
```

Comment:
```python
versions == [int(exploration_version)]
```

Reply:
@lilithxxx is this needed to be resolved?

Reply:
This code is deleted so nothing to resolve.